atlas/AtlasMl/atlasml/ml/pipeline_workflows.py

```
@@ -340,59 +257,157 @@
+            matrix=exercise_embeddings,
+            n_clusters=len(competencies)
+        )
+
+        # Expose clusters
+        for index in range(len(centroids)):
+            cluster = {"cluster_id": uuid.uuid4(), "label_id": str(index), "course_id": course_id}
+            self.weaviate_client.add_embeddings(
+                CollectionNames.CLUSTERCENTER.value, centroids[index].tolist(), cluster
+            )
+
+        clusters = self.weaviate_client.get_all_embeddings(CollectionNames.CLUSTERCENTER.value)
```

Ruff (0.12.2)

Line too long (95 > 88)

(E501)